### DISCUSSION TITLE
Some sorting option for Email Templates

### DISCUSSION THREAD

**Discussion by Thomaswaldick** [emailjs-com/help#54] [Ideas]
It seems like all the email templates just appear in a list in the order they're created which isn't that helpful and can make finding a template a bit tedious. I'd much prefer having the option to have them sorted alphabetically

**Comment by xr0master** [comment]
Thank you for your idea and for making our service better, we really appreciate it!

We thought about this the same way. The idea of ordering by the latest update seemed very convenient since we can immediately see which templates are in development. However, recently this experience has caused us doubts because the templates “jump” down, and it can confuse our users.

I think we will make these changes soon and see how users react.

**Comment by xr0master** [comment]
We have released a new version where we changed the sorting method.
